time_stamper_ros/cfg/Led.cfg

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+PACKAGE = "time_stamper_ros"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+gen.add("frequency",    int_t,    0, "Clock frequency", 50,  1, 10526315)
```

Comment:
hmm I wonder if it makes sense to change frequency completely freely as an integer, as usually not all frequencies can be achieved by the clock.  Maybe a divisor makes more sense? Depends on the use case, though.
Make sure the default, min and max are correct.

Reply:
Hm, this is difficult problem indeed. A divider may also be hard to work with for some scrip not knowing very well about the hardware. One possible way to deal with this is on the node side to change the requested value to a closest possible frequency (see my comment below for how to do that). For interactive configuration this should work well. The user would effectively only be able to adjust the value to something possible. And scripts may also be happy with that. If they really need to know precisely, they could read the effective value back and work with that.